components/bl-autocomplete-component/src/helpers/hooks/useOnClickOutside.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+
+export const useOnClickOutside = (ref, handler) => {
+  useEffect(
+    () => {
+      const listener = event => {
+        if (!ref.current || ref.current.contains(event.target)) {
```

Comment:
```
if (!ref.current || ref.current.contains(event.target)) {
  return;
}

handler(event);
```

to

```
if (ref.current && !ref.current.contains(event.target)) {
  handler(event);
}
```

?

Reply:
Unfortunately, this cannot be done, it will break the work of the hook.

Reply:
I'm not sure I understand on how it might break the hook, could you please explain it with more details